### ISSUE TITLE
Feature: at-least-one-scenario rule

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#159]
I've seen this code in `no-background-only-scenario`:

``` javascript
if (feature.children.length <= 2) {
  // as just one background is allowed, if there is a background in the feature,
  // there must be at least, three elements in the feature to have, more than
  // one scenario
  errors.push(createError(child));
}
```

I'm worried about condition is `feature.children.length <= 2` instead of `feature.children.length === 2`.

Then, when the rule has only a backgrount but no scenarios, it's reporting error about `no-background-only-scenario`. I think it's not the responsability of this rule. We need another rule `at-least-one-scenario`.

**Comment by xgbuils** [comment]
I've seen that the rule that I'm proposing is the same as no-files-without-scenarios. I will update the title because I think that no scenarios should not be reported by `no-background-only-scenario`
